util/container/Dockerfile

```diff
@@ -0,0 +1,45 @@
+#---------------------
+# Ubuntu 22.04 base
+#---------------------
+FROM ubuntu:22.04
+
+#---------------------
+# Install necessary packages
+#---------------------
+RUN apt-get -y update
+RUN apt-get -y install git
+RUN apt-get -y install autoconf help2man perl python3 make flex bison g++ 
+RUN apt-get -y install libfl2  libfl-dev
+RUN apt-get -y install python3 pip
+
+#---------------------
+# Compile verilator from source code
+#---------------------
+RUN git clone https://github.com/verilator/verilator
+WORKDIR verilator/
+RUN git checkout v5.006
+ENV VERILATOR_ROOT=/verilator
+RUN autoconf
+RUN ./configure
+RUN make -j `nproc`
+RUN make test
+ENV PATH="$VERILATOR_ROOT/bin:$PATH"
+
+#---------------------
+# Installing cocotb
+#---------------------
+WORKDIR tmp/
+COPY requirements.txt .
+RUN pip install -r requirements.txt
+
+#---------------------
+# Importing the hwpe-stream
+#---------------------
+WORKDIR /
+RUN git clone https://github.com/KULeuven-MICAS/hwpe-stream.git
+# TODO: fix me later
+WORKDIR /hwpe-stream
+RUN git checkout add-cocotb-test
+RUN git pull
```

Comment:
I would not do it like this. What you are doing right now is this:
1. install all binary dependencies through apt
2. installing source deps through git and make
3. installing python binary dependencies through pip
4. cloning the repo inside your container.

What typically happens in CI is you do all steps until step 3, and you perform step 4 inside the container for each job. Otherwise you have to rebuild the container for each new pull request/job.
In gitlab this is done implicitly with each job, in github actions you use the [checkout action](https://github.com/actions/checkout) for this

Reply:
I updated the README.md with the latest commit [2df74ae](https://github.com/KULeuven-MICAS/hwpe-stream/pull/5/commits/2df74ae5849fff5ff18079c7f03eec0865b02be4). Also I tested the mount, it works! :D